README.md

```diff
@@ -31,7 +31,8 @@ Documentation is written using [VuePress](https://vuepress.vuejs.org/).
 
 1. Clone the repo
 2. Run `yarn install`
-3. Run `yarn docs:dev`
+3. Run `yarn docs:import`
```

Comment:
Hm, I thought that after latest @alexeyzimarev's changes it should work without this step 🤔

Reply:
well it does not, 
I always get that because I'm using `git worktree` , so any branch I check out is a real fresh one.